### PR TITLE
refactor(cloudflare-kv, cloudflare-r2): move `getBindings` to utils and add default `BUCKET` for r2

### PR DESCRIPTION
**Pull request by oritwoen**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The `getBindings` function was used in 2 files. This PR:

- moves the `getBindings` function to `utils`
- adds default `binding` value for `R2`
- adds additional references in the documentation

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/drivers/utils/index.ts [hunk 1]
```diff
@@ -39,3 +40,31 @@ export function createRequiredError(driver: string, name: string | string[]) {
   }
   return createError(driver, `Missing required option \`${name}\`.`);
 }
+
+export function getBinding(binding: KVNamespace | R2Bucket | string) {
```
Comment (pi0) [inline src/drivers/utils/index.ts:44]: Can you please move this to `utils/cloudflare.ts`? 🙏🏼

Comment (oritwoen) [reply src/drivers/utils/index.ts]: Sure, good idea. I have made changes :)

**Comment by pi0** [comment]
(sorry for delay on review. i will do it asap in next days to release 👍🏼 )

**Comment by oritwoen** [comment]
@pi0 

I've been thinking about additional changes lately. There will be more HTTP drivers for Cloudflare in the future. It would be good to put general functions and types in one place:

- https://github.com/unjs/unstorage/blob/main/src/drivers/cloudflare-kv-http.ts#L94
- https://github.com/unjs/unstorage/blob/main/src/drivers/cloudflare-kv-http.ts#L95

etc.

Creating a new PR at this point seems "problematic" because an identical file would be created as in this PR. So maybe change the title of this request and include generalized http functions for cloudflare here as well?

**Comment by oritwoen** [comment]
@pi0 Can you address my comment above?

**Comment by pi0** [comment]
Thanks again! I have made few more refactors to make types working.

PR also welcome to prepare types refactoring we might make `utils/cloudflare.types.ts` to make them more maintainable 💯

**Pull request merged**